x/stakeibc/keeper/hooks.go

```diff
@@ -110,15 +75,7 @@ func (k Keeper) BeforeEpochStart(ctx sdk.Context, epochInfo epochstypes.EpochInf
 	}
 }
 
-func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochInfo epochstypes.EpochInfo) {
-	// every epoch
-	epochIdentifier := epochInfo.Identifier
-	epochNumber := epochInfo.CurrentEpoch
-	k.Logger(ctx).Info(fmt.Sprintf("Handling epoch end %s %d", epochIdentifier, epochNumber))
-	if epochIdentifier == "day" {
-		k.Logger(ctx).Info(fmt.Sprintf("Day %d Ending", epochNumber))
```

Comment:
why remove these logs? are they unnecessary bc we can detect a day ending by the beginning of the next day?

Reply:
yeah exactly, it's just printed immediately before the epoch start, so they're kinda pointless